src/components/common/FilterButtons.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+export default function FilterButtons({
+    cols = 2,
+    filterName = '',
+    selectedValues = [],
+    updateFunc = () => null,
+    values = [],
+} = {}) {
+    const dispatch = useDispatch();
+
+    const toggleValue = value => {
+        const isSelected = selectedValues.includes(value);
+
+        if (isSelected)
+            dispatch(
+                updateFunc({
+                    field: filterName,
+                    value: selectedValues.filter(val => val !== value),
+                })
+            );
+        else dispatch(updateFunc({ field: filterName, value: [...selectedValues, value] }));
```

Comment:
fyi I am going to turn on eslint errors for one-liner if/else statements 😄 https://eslint.org/docs/2.0.0/rules/curly